app/javascript/src/Navigation/components/header/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { PageHeader } from '@patternfly/react-core'
+import Avatar from 'Navigation/components/header/Avatar'
+import Toolbar from 'Navigation/components/header/Toolbar'
+import Brand from 'Navigation/components/header/Brand'
+
+const Header = ({logoProps, toolbarProps, docsProps, avatarProps}) => (
+  <PageHeader
+    logo={<Brand/>}
```

Comment:
OCD triggered!: spaced or not spaced, but always the same.

Reply:
(🙋‍♂️Not spaced please)